Makefile

```diff
@@ -0,0 +1,41 @@
+FAKTORY_HOST=127.0.0.1
+FAKTORY_PORT=7419
+FAKTORY_PORT_UI=7420
+
+check:
```

Comment:
I think this and the other targets that don't produce a file should probably be declared as [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html).

Reply:
With `README.md` as a target - `.PHONY` list will be even required I guess. But even if there were no 'real' targets, explicitly listing phony targets still makes sense due to this (which I honestly never considered before):
```
The implicit rule search ... is skipped for .PHONY targets. This is why declaring a target as .PHONY is good for performance, even if you are not worried about the actual file existing.
```

Thanks for pointing at that. Fixed.

---

Please have a look at `test/perf` and `test/perf/clean`  (instructions taken from  [inferno](https://docs.rs/inferno/latest/inferno/) docs).  I am now working on adding [async](https://github.com/jonhoo/faktory-rs/pull/49) support and, preliminary, I noticed a drop in performance (20-25% less jobs processed per second), which 1) of course can be implementation issue 2) could be even worse if CPU-intensive work were performed in the load test. This is definitely not subject of this PR, but I just realized that having a flamegraph target could be useful.

Reply:
Looks good to me! Could alternatively use `cargo flamegraph` here which does some of this for you, and also works on other platforms.